scripts/accordian.js

```diff
@@ -0,0 +1,76 @@
+    class AccordianComponent {
```

Comment:
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes for how to write classes in es6